src/pages/Settings/Settings.tsx

```diff
@@ -1,33 +1,47 @@
-import { useContext, useEffect, useState } from 'react'
+import { ChangeEvent, useContext, useEffect, useState } from 'react'
 import Box from '@mui/material/Box'
 import Button from '@mui/material/Button'
 import Typography from '@mui/material/Typography'
 import Divider from '@mui/material/Divider'
+import { Switch } from '@mui/material'
 
 import { ShellContext } from 'contexts/ShellContext'
 import { StorageContext } from 'contexts/StorageContext'
 import { PeerNameDisplay } from 'components/PeerNameDisplay'
 
 import { ConfirmDialog } from '../../components/ConfirmDialog'
+import { SettingsContext } from '../../contexts/SettingsContext'
 
 interface SettingsProps {
   userId: string
 }
 
 export const Settings = ({ userId }: SettingsProps) => {
   const { setTitle } = useContext(ShellContext)
+  const { updateUserSettings, getUserSettings } = useContext(SettingsContext)
   const { getPersistedStorage } = useContext(StorageContext)
   const [
     isDeleteSettingsConfirmDiaglogOpen,
     setIsDeleteSettingsConfirmDiaglogOpen,
   ] = useState(false)
+  const [playSoundOnNewMessage, setPlaySoundOnNewMessage] = useState(
+    () => getUserSettings().playSoundOnNewMessage
+  )
 
   const persistedStorage = getPersistedStorage()
 
   useEffect(() => {
     setTitle('Settings')
   }, [setTitle])
 
+  const handlePlaySoundOnNewMessageChange = (
+    event: ChangeEvent,
```

Comment:
`event` isn't used, so it's causing a Typescript warning:

```
'event' is declared but its value is never read. (tsserver 6133)
```

We can silence that warning by prepending an underscore to the variable name:

```suggestion
    _event: ChangeEvent,
```